data/lynis/baseline-lynis-audit-system-nocolors-sle15sp3-x86_64-snapshot7-textmode

```diff
@@ -0,0 +1,630 @@
+
+[ Lynis 2.6.1 ]
+
+################################################################################
+  Lynis comes with ABSOLUTELY NO WARRANTY. This is free software, and you are
+  welcome to redistribute it under the terms of the GNU General Public License.
+  See the LICENSE file for details about using this software.
+
+  2007-2018, CISOfy - https://cisofy.com/lynis/
+  Enterprise support available (compliance, plugins, interface and tools)
+################################################################################
+
+
+[+] Initializing program
+------------------------------------
+[2C- Detecting OS... [41C [ DONE ]
+[2C- Checking profiles...[37C [ DONE ]
+
+  ---------------------------------------------------
+  Program version:           2.6.1
+  Operating system:          Linux
+  Operating system name:     Linux
+  Operating system version:  5.3.18-43-default
+  Kernel version:            5.3.18
+  Hardware platform:         x86_64
+  Hostname:                  susetest
+  ---------------------------------------------------
+  Profiles:                  /etc/lynis/default.prf
+  Log file:                  /var/log/lynis.log
+  Report file:               /var/log/lynis-report.dat
+  Report version:            1.0
+  Plugin directory:          /usr/share/lynis/plugins
+  ---------------------------------------------------
+  Auditor:                   [Not Specified]
+  Language:                  en
+  Test category:             all
+  Test group:                all
+  ---------------------------------------------------
+[2C- Program update status... [32C [ WARNING ]
+
+      ===============================================================================
+        Lynis update available
+      ===============================================================================
+
+        Current version is more than 4 months old
+
+        Current version : 261   Latest version : 303
+
+        Please update to the latest version.
```

Comment:
It describes lynis has the latest version and please update to the latest, should we need to do it?

Reply:
> It describes lynis has the latest version and please update to the latest, should we need to do it?

I think no. Developer will decide if they need to use the latest version.
Actually we had asked developer before he suggested we just use the pkg offered in our product module.

Reply:
okay, got it, noted.